lib/main.js

```diff
@@ -111,6 +111,14 @@ exports.main = function(options, callbacks) {
         ss.storage.meanings = true;
     }
 
+    // if the ATB parm is not defined, the extension is either being installed
+    // or we ran into a strange bug with Firefox v49 which requires a restart
+    // for the install of the first extension and never sends the 'install'
+    // trigger from options.loadReason
+    if (ss.storage.atb == undefined) {
```

Comment:
@bsstoner I do not see how would that happen, but let's be rather safe than sorry =)

Thanks!